Pidaq/UI/Stability/stability_dash/layout/control.py

```diff
@@ -0,0 +1,151 @@
+import dash
+import dash_bootstrap_components as dbc
+import dash_daq as daq
+import dash_core_components as dcc
+import dash_html_components as html
+
+# Main Control GUI Bootstrap Layout
+control_layout = html.Div([
+
+dbc.Row([
+    dbc.Col([
+        dbc.Card([
+            dbc.CardBody(
+                html.H3("PARAMETERS",
+                    style={"text-align": "center"})
+            )
+        ])
+    ],
+    width={"size": 4, "offset": 4})
+]),
+
+
```

Comment:
Should be always one space between rows or always 2 spaces, seems like you went with mostly one space so you can just make this one space as well